translate.js

```diff
@@ -306,6 +317,7 @@ export default (theme) => (str) => {
               out[`list-style-type`] = i[1];
               break;
           }
+          break;
```

Comment:
Unrelated, but my IDE caught this as a potential fall through situation.

Reply:
Humm that is weird.. are we missing a break somewhere?

Reply:
Well not after this addition, no.

Reply:
Ahh yeh.. duhh. Sorry, I thought you were just highlighting the line rather than the addition 😅 thank you.